src/explore-education-statistics-frontend/src/components/DownloadCsvButtonWithAnalytics.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import DownloadCsvButton, {
+  DownloadCsvButtonProps,
+} from '@common/modules/table-tool/components/DownloadCsvButton';
+import { logEvent } from '@frontend/services/googleAnalyticsService';
+
+const DownloadCsvButtonWithAnalytics = (props: DownloadCsvButtonProps) => {
+  return (
+    <DownloadCsvButton
+      {...props}
+      onClick={() =>
+        logEvent(
+          'CSV download',
```

Comment:
Category should probably be 'Table tool' or 'Permalink' depending on which page you're on.